zokrates_ast/src/zir/identifier.rs

```diff
@@ -34,10 +35,17 @@ impl<'ast> fmt::Display for Identifier<'ast> {
     fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
         match self {
             Identifier::Source(s) => write!(f, "{}", s),
+            Identifier::Internal(s) => write!(f, "{}", s),
```

Comment:
We have some compiler-introduced variables in this PR (check other comments for clarification) so I assumed it would be best to introduce a new variant here, as the Source one has a `'ast` lifetime.